kip-0026.md

```diff
@@ -0,0 +1,105 @@
+---
+KIP: 0026
+Title: Key Derivation and Mnemonic encoding methods
+Author: Tasos Bitsios @Takadenoshi
+Status: Draft
+Type: Standard
+Category: Application
+Created: 2023-11-21
+---
+
+# Abstract
+
+We propose the following key derivation and mnemonic encoding methods for Chainweb keys:
+
+- Derivation: SLIP-10 with derivation path `m/44'/626'/a'`.
+- Mnemonics: BIP-39, 24 words, English wordlist, empty passphrase.
+
+# Motivation
+
+Convergence on these standards will ensure key and mnemonic interoperability between wallet providers. This will benefit users of all wallet types (hot/hardware/cold), who will be able to reuse their keys or phrases without requiring conversion.
+
+# Specification
+
+## Key Derivation
+
+We recommend following the widely adopted standard for deriving ED25519 keypairs: [SLIP-0010](https://github.com/satoshilabs/slips/blob/master/slip-0010.md).
```

Comment:
I don't think the KIP itself should be stated as a recommendation. Rather, I'd state directly what anyone following the KIP should follow, and then later we can say to a wallet provider that "We recommend you follow KIP-26".